sdk/monitor/azquery/README.md

```diff
@@ -311,6 +321,22 @@ Response
 |---Resourceregion *string
 ```
 
+#### List Metric Definitions
+
+To list the metric definitions for the resource, use the `NewListDefinitionsPager` method.
+
+```go
+pager := client.NewListDefinitionsPager(resourceURI, nil)
```

Comment:
Snippets should be useful as copy pasta, so paging examples should show the pager API. But it may be best to expand these later, when moving them to testable examples (#18786)

Reply:
Good idea, will expand later when moving to testable examples